src/Repman.php

```diff
@@ -60,10 +61,15 @@ public function populateMirrors(InstallerEvent $installerEvent): void
                 continue;
             }
 
+            /** @var PackageInterface $package */
             if (!method_exists($package, 'setDistMirrors')) {
                 continue;
             }
 
+            if (strstr((string) $package->getNotificationUrl(), 'packagist.org') === false) {
```

Comment:
currently the best option for me, other tools will add own notification url (like @repman-io)